scripts/create-jira-server-symmetric-jwt.ts

```diff
@@ -0,0 +1,54 @@
+import { createQueryStringHash, encodeSymmetric } from 'atlassian-jwt';
+import querystring from 'querystring';
+import { Command, InvalidArgumentError } from 'commander';
+
+const program = new Command();
+
+const intParser = (value: string) => {
+	const parsedValue = parseInt(value, 10);
+	if (isNaN(parsedValue)) throw new InvalidArgumentError('Not a number.');
+
+	return parsedValue;
+};
+
+program
```

Comment:
**note:** This dev script replaces the legacy `create-jwt`:

- Adds additional documentation and error handling
- Makes obvious which type of a JWT token is generated.

The script is implemented using [Commander](https://github.com/tj/commander.js#readme).